site/content/docs/playwright-checks/_index.md

```diff
@@ -49,106 +46,53 @@ What you need:
   npm install --save-dev jiti typescript
   ```
 
-### 3. Define which tests to monitor in `checkly.config.ts/js`
-
-Create a `checkly.config.ts/js` and pick the tests you want to monitor, use [test tags](https://playwright.dev/docs/test-annotations#tag-tests) and [projects](https://playwright.dev/docs/test-projects) to create a Playwright Check Suite.
-
-Make sure:
-
-* `pwProjects` match projects names in your `playwright.config.ts`.
-* `pwTags` match tags in your test definitions.
-
-Below are two example check suites:
-
-* a full multi-browser suite with smoke tests.
-* a `chromium`-only check suite running tests tagged with `@critical`.
-
-```typescript {title="checkly.config.ts/js"}
-import { defineConfig } from 'checkly'
-import { Frequency } from 'checkly/constructs'
-
-export default defineConfig({
-  projectName: 'Cool Website Checks',
-  logicalId: 'cool-website-monitoring',
-  repoUrl: 'https://github.com/acme/website',
-  checks: {
-    playwrightConfigPath: './playwright.config.ts',
-    playwrightChecks: [
-      {
-        /**
-         * Create a multi-browser check that runs
-         * every 10 mins in two locations.
-         */
-        logicalId: 'multi-browser',
-        name: 'Multi Browser Suite',
-        // Use one project (or multiple projects) defined in your Playwright config
-        pwProjects: ['chromium', 'firefox', 'webkit'],
-        // Use one tag (or multiple tags) defined in your spec files
-        pwTags: '@smoke-tests',
-        frequency: Frequency.EVERY_10M,
-        locations: ['us-east-1', 'eu-west-1'],
-      },
-      {
-        /**
-         * Create a check that runs the `@critical` tagged tests
-         * every 5 mins in three locations.
-         */
-        logicalId: 'critical-tagged',
-        name: 'Critical Tagged tests',
-        // Use one project (or multiple projects) defined in your Playwright config
-        pwProjects: ['chromium'],
-        // Use one tag (or multiple tags) defined in your spec files
-        pwTags: '@critical',
-        frequency: Frequency.EVERY_5M,
-        locations: ['us-east-1', 'eu-central-1', 'ap-southeast-2'],
-      },
-    ],
-  },
-  cli: {
-    runLocation: 'us-east-1',
-    retries: 0,
-  },
-})
-```
 
-Access [the Playwright Check Suites reference documentation](/docs/playwright-checks/reference/) for more configuration options.
 
-### 4. Validate your monitors
+### 3. Use `pw-test` for quick testing
 
-Validate the Playwright check suites that reference existing Playwright tags or projects in your repository by running `npx checkly test` from your terminal. Your Playwright check suites will then be executed in the Checkly infrastructure running in the location specified in your Checkly config (`cli.runLocation`).
+Let's start with confirming your tests will work on Checkly, run the ones you want using `pw-test`.
```

Comment:
This feels a little thin. Should we lean into the narrative of running your entire test suite in CI/CD and deploy only a subset? "Quick testing" feels a little weak to me.